frontend/src/components/CanteenPublication.vue

```diff
@@ -75,12 +75,27 @@
         {{ canteen.publicationComments }}
       </p>
     </div>
+
+    <div v-if="canteen && graphDiagnostics">
+      <h2 class="font-weight-black text-h6 grey--text text--darken-4 mt-12 mb-2">
+        L'évolution de l'approvisionnement
+      </h2>
+      <p class="body-2">
```

Comment:
Pourquoi est-ce que tu utilises le <p> ici ?

Reply:
No good reason, I'll change it to a div

Reply:
Done here : https://github.com/betagouv/ma-cantine/pull/581/commits/6fc51a52befe7b088ee4d4430a1292470a03f906